### DISCUSSION TITLE
/devhack-open-vscode-extension-settings-code/

### DISCUSSION THREAD

**Discussion by comment-migrator** [estruyf/eliostruyf-comments#552] [Comments]
#DevHack: Open your VSCode extension settings from the code
        
[https://www.eliostruyf.com/devhack-open-vscode-extension-settings-code/](https://www.eliostruyf.com/devhack-open-vscode-extension-settings-code/)

**Comment by ilyabelikin** [comment]
Thanks! This was very helpful.
